src/components/context/DataProvider.js

```diff
@@ -48,6 +48,14 @@ const DataProvider = ({ children }) => {
     },
     { enabled: Boolean(enabledArray[Context.EXPLORER]) },
   );
+  const { data: unknownData, isError: unknownIsError } = hooks.useActions(
+    {
+      itemId,
+      view: 'unknown',
```

Comment:
I know we did it together quite quickly, but I suggest to add it to the `Context` object so it can be easily reused.